site/app/templates/polls/ViewPoll.twig

```diff
@@ -76,18 +76,23 @@
                                    onchange="disableNoResponse();"
                             />
                             </td>
-                            <td class="option-text">
+                            <td class="option-text" id = "markdown-{{index}}">
                             {% include "misc/Markdown.twig" with {
                                 "content" : option.getResponse(),
                                 "style": "display: inline"
                             } only %}
+                            {% if ((poll.getAllowsCustomResponses() and option.isCustom()) and (user_id == option.getAuthorId())) %}
+                                <div class="move-btn delete-btn" style = "border:0; padding:0;" onclick = "removeCustomResponse({{ poll.getId() }}, {{ option.getId() }}, '{{ base_url }}');">
```

Comment:
Is this styling necessary?  This seems like a good place to use a button, rather than a div which acts like a button.

Reply:
For some reason, making it a button instead of a div causes the user to exit the poll after the click. Working on a solution

Reply:
Changed to button in [73ff75e](https://github.com/Submitty/Submitty/pull/10197/commits/73ff75e05051e841621aa6cf27980d35bd10874b)